namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/03-resourcequota.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: namespace-quota
+  namespace: pathfinder-dev
+spec:
+  hard:
+    requests.cpu: 3000m
```

Comment:
This will reserve 3 CPU cores, as soon as this namespace is created. This is unlikely to be useful, and causes resource contention in the cluster. 
Please reduce this to 100m, as per our template: https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespace-resources/03-resourcequota.yaml
NB: This does not limit your namespace in any way, it merely tells the cluster how much resource to *reserve* for your namespace